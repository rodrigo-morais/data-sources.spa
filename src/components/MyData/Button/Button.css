button {
  background-color: #E67E22;
  color: #ffffff;
  height: 50%;
  width: 38%;
  place-self: center end;
  margin-right: 6%;
  border-radius: 20px;
  background-image: linear-gradient(to right, #E67E22, #E67E22 40%, #EB984E 65%, #F0B27A 80%, #F5CBA7 95%);
  font-weight: bold;
  min-width: 140px;
}
