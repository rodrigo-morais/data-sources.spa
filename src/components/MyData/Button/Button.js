import Vue from 'vue';
import Component from 'vue-class-component';

export default Component({
  name: 'Button',
})(class Buttonr extends Vue {
  title = 'ADD DATA SOURCE'
});
