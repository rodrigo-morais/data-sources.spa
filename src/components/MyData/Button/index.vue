<template src='./Button.html'></template>

<script src='./Button.js'></script>

<style scoped src='./Button.css'></style>
