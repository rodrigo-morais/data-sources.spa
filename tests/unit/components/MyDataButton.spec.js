import { shallowMount } from '@vue/test-utils'
import { createRenderer } from 'vue-server-renderer'
import Button from '@/components/MyData/Button/index.vue'

describe('MyData.Button.vue', () => {
  it('renders according to design', () => {
    const renderer = createRenderer()
    const wrapper = shallowMount(Button)
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err)
      expect(str).toMatchSnapshot()
    })
  })
})
