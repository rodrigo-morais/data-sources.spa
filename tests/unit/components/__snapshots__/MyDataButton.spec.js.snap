// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyData.Button.vue renders according to design 1`] = `
<button>
  ADD DATA SOURCE
</button>
`;
